### PR TITLE
CodeCamp #43 Support PETR in 1.1

### PR DESCRIPTION
**Pull request by SekiroRong**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ZwwWayne** [comment]
We do not need to have so many directories in projects/petr/petr, we only need to keep one and put all files into it.

File: configs/petr/petr_vovnet_gridmask_p4_800x320.py [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+    'motorcycle', 'bicycle', 'pedestrian', 'traffic_cone'
+]
+metainfo = dict(classes=class_names)
+
+input_modality = dict(
+    use_lidar=False,
+    use_camera=True,
+    use_radar=False,
+    use_map=False,
+    use_external=True)
```
Comment (ZCMax) [inline configs/petr/petr_vovnet_gridmask_p4_800x320.py:29]: Why do we need `use_external`?

File: configs/petr/petr_vovnet_gridmask_p4_800x320.py [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+model = dict(
+    type='Petr3D',
```
Comment (ZCMax) [inline configs/petr/petr_vovnet_gridmask_p4_800x320.py:31]: Petr3D -> PETR

File: configs/petr/petr_vovnet_gridmask_p4_800x320.py [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+    dict(
+        type='ResizeCropFlipImage', data_aug_conf=ida_aug_conf, training=True),
+    dict(
+        type='GlobalRotScaleTransImage',
+        rot_range=[-0.3925, 0.3925],
+        translation_std=[0, 0, 0],
+        scale_ratio_range=[0.95, 1.05],
+        reverse_angle=False,
+        training=True),
+    dict(type='PadMultiViewImage', size_divisor=32),
```
Comment (ZCMax) [inline configs/petr/petr_vovnet_gridmask_p4_800x320.py:188]: padding should be down in `data_preprocessor`

File: configs/petr/petr_vovnet_gridmask_p4_800x320.py [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+        ])
+]
+test_pipeline = [
+    dict(type='LoadMultiViewImageFromFiles', to_float32=True),
+    dict(type='AddCamInfo'),
+    dict(
+        type='ResizeCropFlipImage', data_aug_conf=ida_aug_conf,
+        training=False),
+    # TODO Figure whether need Pad
+    dict(type='PadMultiViewImage', size_divisor=32),
```
Comment (ZCMax) [inline configs/petr/petr_vovnet_gridmask_p4_800x320.py:203]: please check here whether padding is needed

**Pull request closed without merging**